packages/combo-box/src/vaadin-combo-box-mixin.js

```diff
@@ -203,8 +203,8 @@ export const ComboBoxMixin = (subclass) =>
     static get observers() {
       return [
         '_filterChanged(filter, itemValuePath, itemLabelPath)',
-        '_itemsOrPathsChanged(items.*, itemValuePath, itemLabelPath)',
-        '_filteredItemsChanged(filteredItems.*, itemValuePath, itemLabelPath)',
+        '_itemsOrPathsChanged(items, itemValuePath, itemLabelPath)',
+        '_filteredItemsChanged(filteredItems, itemValuePath, itemLabelPath)',
```

Comment:
I don't see the need to pass `itemValuePath` and `itemLabelPath` as dependencies to these observers.

When removing them, no tests fail. Also on master the observer only runs on `filteredItems` change.

```suggestion
        '_filteredItemsChanged(filteredItems)',
```

Reply:
Done.